shared/utils.py

```
@@ -92,6 +92,46 @@ def get_os_text(key: str) -> str:
+
+    if filepath.lower().endswith('.pdf'):
+        try:
+            import fitz  # PyMuPDF
+            doc = fitz.open(filepath)
+            try:
+                if doc.page_count > 0 and _PO_RFQ_TEXT_RE.search(doc[0].get_text()):
+                    return True, "PDF content contains PO/RFQ keyword"
+            finally:
+                doc.close()
+        except Exception:
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)